syncer/ddl.go

```diff
@@ -138,6 +154,29 @@ func (s *Syncer) splitAndFilterDDL(
 
 		// get real tableNames before apply block-allow list
 		if s.onlineDDL != nil {
+			if onlineDDLFinish && len(tables) > 1 {
```

Comment:
```suggestion
			if onlineDDLFinish && len(tables) = 2 {
```

Reply:
Maybe we can remove `len(tables) > 1`, because after `FetchDDLTables` `ast.RenameTableStmt`, len(tables) must equal 2 which test in https://github.com/pingcap/dm/blob/master/pkg/parser/common_test.go#L136-L142.